src/packages.builds

```diff
@@ -10,7 +10,7 @@
     <Project Include="Native\pkg\**\*.builds">
```

Comment:
I would add a "SkipNativePackageBuild" property condition for symmetry.  I understand there is no immediate need for this property at the moment though.